src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -530,6 +562,84 @@ public bool IsLexicallyContainedInMethod(MethodInfo containerMethod)
                     (EndLocation.Line == containerMethod.EndLocation.Line && EndLocation.Column < containerMethod.EndLocation.Column));
     }
 
+    internal class ParameterInfo
+    {
+        public string Name { get; private set; }
+
+        public bool HasDefaultValue { get; private set; }
```

Comment:
Do we really need this separately? How about we remove this, and rename `Value` to `DefaultValue`? The caller can easily check `DefaultValue != null` to see if there is a valid or not.